### PR TITLE
Added a missing step in Hacking.md

### PR DESCRIPTION
**Pull request by Tanesh1701**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

**Comment by kyrofa** [comment]
This is a bug, right? Can you add a link to the bug in the PR description?

**Comment by Tanesh1701** [comment]
This change is for a missing step in the guide Hacking.md found on this URL: https://github.com/snapcore/snapcraft/blob/master/HACKING.md

File: HACKING.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ Create and activate a new virtual environment:
 
 Make sure pip is up-to-date:
 
-    (snapcraft) $ pip install --upgrade pip
+   (snapcraft) $ pip install --upgrade pip
+    
+Change directory to Snapcraft:
+
+   (snapcraft) $ cd snapcraft
```
Comment (elopio) [inline HACKING.md]: oh, but we are never mentioning here about the clone.
What about mentioning to fork the github repo, and give the command to clone your fork?

File: HACKING.md [hunk 2]
```diff
@@ -5,6 +5,12 @@
 First install a few dependencies:
 
     sudo apt install gcc g++ make python3-dev python3-venv libffi-dev libsodium-dev libapt-pkg-dev libarchive13 squashfs-tools patchelf
+    
+   
+Fork the repository snapcraft on https://github.com/snapcore/snapcraft
```
Comment (kyrofa) [inline HACKING.md]: I'm not sure I agree with @elopio, here. This is in the "installing from pip" section. The "installing for development" section is below. Telling people to fork here seems premature. I'd rather see it below.

Comment (elopio) [reply HACKING.md]: As long as it's before the call that uses requirements.txt, it works for me.

File: HACKING.md [hunk 3]
```diff
@@ -15,7 +21,12 @@ Create and activate a new virtual environment:
 
 Make sure pip is up-to-date:
 
-    (snapcraft) $ pip install --upgrade pip
+   (snapcraft) $ pip install --upgrade pip
```
Comment (kyrofa) [inline HACKING.md]: Please also fix your indentation, here.

**Comment by sergiusens** [comment]
looks abandoned. If you ever update it or think this is a mistake please do not hesitate to raise this.

**Pull request closed without merging**